templates/user/dashboard/issues.tmpl

```diff
@@ -74,7 +74,7 @@
 					<form class="list-header-search ui form ignore-dirty">
 						<div class="ui small search fluid action input">
 							<input type="hidden" name="type" value="{{$.ViewType}}">
-							<input type="hidden" name="repos" value="[{{range $.RepoIDs}}{{.}}%2C{{end}}]">
+							<input type="hidden" name="repos" value="[{{range $.RepoIDs}}{{.}},{{/*It should be "," not "%2C", because it's a form input not a url parameter*/}}{{end}}]">
```

Comment:
```suggestion
							<input type="hidden" name="repos" value="[{{range $.RepoIDs}}{{.}},{{end}}]">
```

I do not think it needs comment ..... This value is in a form, is will always be escaped by the browser correctly.